public/js/components/SplitBox.css

```diff
@@ -14,7 +14,7 @@
 .split-box .controlled {
   display: flex;
   overflow: auto;
-  transition: width 0.5s ease;
+  /*transition: width 0.5s ease;*/
```

Comment:
This fixes resizing with the mouse for chrome. Firefox is still pretty fussy, but maybe reducing the easing to a smaller number might help